schemas/org.gnome.shell.extensions.paperwm.gschema.xml

```diff
@@ -238,6 +238,14 @@
         <default><![CDATA[[]]]></default>
         <summary>Switch to the below window (no monitor boundary)</summary>
     </key>
+    <key type="as" name="switch-up-or-else-workspace">
+        <default><![CDATA[['']]]></default>
```

Comment:
@jtaala should this be `<![CDATA[['']]]>`or `<![CDATA[[]]]>`? It looks like we used it inconsistently.

Reply:
Good pickup @Lythenas - I don't think it matters here much(?), but for consistency and intent, it should be the latter for no keybind applied.